### PR TITLE
Detect new brands: AAUW, Aocwei, Rocket, SMUX and Detect devices for exist brands

### PR DESCRIPTION
**Pull request by sanchezzzhak**

### CODE REVIEW

File: regexes/device/mobiles.yml [hunk 1]
```diff
@@ -40506,6 +40579,30 @@ ANBERNIC:
...
+  model: 'A1'
+
+# AAUW (aauwtablet.com)
+AAUW:
+  regex: 'Android 13(?:[\d.]*);(?: [\w-]+;)? (?:M50)( Build|[;)])'
+  device: 'tablet'
+  model: 'M50'
+
+# Rocket Tablet Computer
+Rocket:
```
Comment (sanchezzzhak) [inline regexes/device/mobiles.yml:40601]: is it worth specifying the full name as on the cover of the device?
<details>
<summary>Preview image</summary>

image

</details>

Comment (liviuconcioiu) [reply regexes/device/mobiles.yml:40601]: I think it is. We already have some full names and also will avoid confusion for further detections.

File: regexes/device/mobiles.yml [hunk 2]
```diff
@@ -39165,7 +39238,7 @@ iRobot:
 
 # BRAVE
 BRAVE:
-  regex: '(BT7X1)(?:[);/ ]|$)'
+  regex: '(BT7X1|EX10SL4)(?:[);/ ]|$)'
```
Comment (liviuconcioiu) [inline regexes/device/mobiles.yml:39241]: Can you add a comment for `EX10SL4`, `# EXCEED EX10SL4`?

File: Tests/fixtures/smartphone-36.yml [hunk 3]
```diff
@@ -5315,7 +5315,7 @@
     engine_version: 79.0.3945.93
   device:
     type: smartphone
-    brand: Freetel
+    brand: Plusone
```
Comment (Simbiat) [inline Tests/fixtures/smartphone-36.yml:5318]: FTJ161E-VN seems to be Freetel, though, based on search. Why the change?

File: Tests/fixtures/smartphone-39.yml [hunk 4]
```diff
@@ -4709,3 +4709,867 @@
...
+    platform: ""
+  client:
+    type: browser
+    name: Chrome Mobile
+    version: 71.0.3578.99
+    engine: Blink
+    engine_version: 71.0.3578.99
+  device:
+    type: smartphone
+    brand: Plusone
```
Comment (Simbiat) [inline Tests/fixtures/smartphone-39.yml:4906]: Are we sure it's the brand? Nothin in user_agent indicates this. and it would make more sense if it's https://www.imei.info/phonedatabase/orbic-funplus/ if it's a phone even or https://apkpure.net/fun-plus-xc/com.nnetwork.funplus

Comment (sanchezzzhak) [reply Tests/fixtures/smartphone-39.yml:4906]: orbic fun+  android OS 12 version

https://www.gsmchoice.com/en/catalogue/orbic/funplus4g/

Comment (Simbiat) [reply Tests/fixtures/smartphone-39.yml:4906]: which means the brand is incorrect, no?

Comment (sanchezzzhak) [reply Tests/fixtures/smartphone-39.yml]: brand, Freetel was founded by Plus One Marketing in 2012
https://ja.wikipedia.org/wiki/FREETEL

I returned the name back, also deleted PlusOne

**Review by liviuconcioiu — APPROVED** [review]
